quality-assurance/performance-test-checklist.md

```diff
@@ -0,0 +1,80 @@
+# Performance testing checklist
+
+## Purpose of this document
```

Comment:
I am not sure how this gets through the linter - you need new lines after all headings.

Reply:
done